Projects/Modules/Documentarian.ModuleAuthor/CHANGELOG.md

```diff
@@ -30,6 +30,13 @@ Related Links
 
 ### Added
 
+- Update `Get-ParameterInfo` and `Find-ParameterWithAttribute` to support for three more parameter
+  attributes:
+
+  - CredentialAttribute
+  - ObsoleteAttribute
+  - PSDefaultValueAttribute
```

Comment:
```suggestion
- Update `Get-ParameterInfo` and `Find-ParameterWithAttribute` to support three more parameter
  attributes:

  - {{% xref "System.Management.Automation.CredentialAttribute" %}}
  - {{% xref "System.Management.Automation.PSDefaultValueAttribute" %}}
  - {{% xref "System.ObsoleteAttribute" %}}
```

Reply:
Minor change, this will ensure those automatically link to the dotnet docs for those attributes to provide a bit of context.